src/apps/config/leader.lua

```diff
@@ -70,16 +69,31 @@ end
 
 function Leader:set_initial_configuration (configuration)
    self.current_configuration = configuration
-   self.current_app_graph = self.setup_fn(configuration)
    self.current_in_place_dependencies = {}
-   self.current_in_place_dependencies =
+
+   -- Start the followers and configure them.
+   local follower_configurations = self.setup_fn(configuration)
+
+   -- Calculate the dependences
+   self.in_place_dependencies =
```

Comment:
Why rename from `current_in_place_dependencies`?  What's the difference between this one and the one defined above?